examples/storybook/src/custom-widgets.ts

```diff
@@ -37,7 +37,7 @@ export class MenuSelect extends BaseWidget implements OnInit {
   }
 
   public ngOnInit() {
-    this.createWidget(connectMenu, { attribute: 'categories' });
+    this.createWidget(connectMenu as Connector, { attribute: 'categories' });
```

Comment:
what's the cast needed for? doesn't it work with -experimental-typescript now?

Reply:
I'm not quite sure how to solve this one yet, but it's essentially coming from the fact BaseWidget is expecting `Connector` declared [here](https://github.com/algolia/angular-instantsearch/commit/c3c7a6d3981eaa1bca84ccab06fcbd4b4eef8055#diff-6e7f95676ef69841ddac5acb1079866545e94ad8ddea3e1efe2cccccb382a987R11) whereas `connectMenu` is a different type `Connector<TRendererOptions, TConnectorParams>`. 
This is the quickest way I found to make this work. Any better ideas?

Reply:
I think BaseWidget should change to make sure it accepts InstantSearch connectors, but I think we can do it after this PR, fine enough for now. It should probably not reassign connector, but import from InstantSearch.js

Reply:
I guess that would require exposing `TRendererOptions` and `TConnectorParams` outside `BaseWidget` (or there another way ?)
For me changing BaseWidget implem, while need to v4, will impact all the widgets + the docs: so deserves its own PR for me.